ThreeCardMonte/ThreeCardMonte/ViewController.swift

```diff
@@ -10,11 +10,54 @@ import UIKit
 
 class ViewController: UIViewController {
 
-  override func viewDidLoad() {
+    
+    
+    @IBOutlet weak var cardThreeOutlet: UIButton!
+    
+    
+    
+    @IBOutlet weak var cardTwoOutlet: UIButton!
+    
+    
+    
+    @IBOutlet weak var cardOneOutlet: UIButton!
+    
+    
+    
+    
+//    var threeCards = [UIButton]
+    
+    @IBAction func resetGame(_ sender: UIButton) {
```

Comment:
If this function restarts the game, all the cards should be assigned to the "back of the card" image.